src/main/java/uk/gov/hmcts/reform/adoption/notification/SendgridService.java

```diff
@@ -95,6 +98,7 @@ public void sendEmail(CaseData caseData, String subject, DocumentType documentTy
             log.info("Uploaded Documents size:  {}", uploadedDocumentsUrls.size());
             for (AdoptionDocument item : uploadedDocumentsUrls) {
                 String url = StringUtils.substringAfterLast(item.getDocumentLink().getUrl(), "/");
+                log.info("About to call getDocumentBinary method to fetch uploaded document(s) binary");
```

Comment:
Added the comments as we are calling a third party API to fetch the document binary and hence sometimes it fails and with these logs it would be easier to track any failures.